src/arb_types.jl

```diff
@@ -59,26 +59,41 @@ mutable struct arb_struct
 end
 
 mutable struct acb_struct
-                          # ┌ arb_struct (real)
+    # ┌ arb_struct (real)
     exponent_r::UInt      # │ fmpz
     size_r::UInt          # │ mp_size_t
     mantissa1_r::UInt     # │ mantissa_struct of length 128
     mantissa2_r::UInt     # │
     exp_mag_r::Int        # │ fmpz
     mantissa_mag_r::UInt  # │ mp_limb_t
-                          # └
-                          # ┌ arb_struct (imag)
+    # └
+    # ┌ arb_struct (imag)
     exponent_i::Int       # │ fmpz
     size_i::UInt          # │ mp_size_t
     mantissa1_i::UInt     # │ mantissa_struct of length 128
     mantissa2_i::UInt     # │
     exp_mag_i::Int        # │ fmpz
     mantissa_mag_i::UInt  # │ mp_limb_t
-                          # └
+    # └
     function acb_struct()
         res = new()
         init!(res)
         finalizer(clear!, res)
         return res
     end
 end
+
+mutable struct acb_mat_struct
+
+    entries::Ptr{acb_struct}
+    r::Clong
+    c::Clong
+    rows::Ptr{Ptr{acb_struct}}
+
+    function acb_mat_struct(n::Integer, m::Integer)
```

Comment:
I would propose to use `r` and `c` for rows and columns to follow the convention in Arb. Same for the `AcbMatrix` constructor (which currently uses `n` and `m` in the opposite way).

Reply:
good point, I will make this consistent